admin/server/organizations.go

```diff
@@ -664,7 +664,7 @@ func (s *Server) GetOrganizationMemberUser(ctx context.Context, req *adminv1.Get
 	}
 
 	// Find the organization member
-	members, err := s.admin.DB.FindOrganizationMemberUsers(ctx, org.ID, "", true, user.Email, 1, "")
+	members, err := s.admin.DB.FindOrganizationMemberUsers(ctx, org.ID, "", true, "", 1, user.Email)
```

Comment:
It's not safe to rely on a search pattern for a single-row lookup. For example, emails can have a `%` in them. Maybe add a proper singular `FindOrganizationMemberUser` DB query?